src/lib.rs

```diff
@@ -25,6 +24,8 @@ mod mysql;
     feature = "db-diesel-postgres",
 ))]
 mod postgres;
+#[cfg(feature = "rand")]
```

Comment:
Thank you - I do like seeing the flags here to be consistent.

Reply:
Yeah, the inconsistency irked my OCD. :-)